src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -187,6 +187,10 @@ protected void LookupMembersInType(LookupResult result, TypeSymbol type, string
                     this.LookupMembersInClass(result, type, name, arity, basesBeingResolved, options, originalBinder, diagnose, ref useSiteDiagnostics);
                     break;
 
+                case TypeKind.Tuple:
+                    useSiteDiagnostics = LookupMembersInTuple(result, type, name, arity, basesBeingResolved, options, originalBinder, diagnose, useSiteDiagnostics);
+                    break;
```

Comment:
I did this through refactoring and did not notice the change.
I am not yet sure if we need a separate LookupMembersInTuple. So far it ended up just deferring to LookupMembersInClass. 
I think we might do the first iteration (before going into bases) in the LookupMembersInTuple and then call into LookupMembersInClass for underlying type, since that would be the same. The only gain would be that the special case check for tuples could be done only in LookupMembersInTuple and not affect other types that are not tuples.
For now I did not want to complicate things though, so LookupMembersInTuple just calls LookupMembersInClass.